spacy/tokens/span.pyx

```diff
@@ -101,6 +101,8 @@ cdef class Span:
             label = doc.vocab.strings.add(label)
         if isinstance(kb_id, str):
             kb_id = doc.vocab.strings.add(kb_id)
+        if isinstance(id, str):
+            id = doc.vocab.strings.add(id)
```

Comment:
I think this code is unideal because `id` is a built-in Python function :/

Reply:
I know, but all the other alternatives for the kwarg here seemed worse?

Reply:
why not `span_id`?

Reply:
I think `Span(span_id=1234)` is a little weird, but I guess that could work.